lib/vision/index.js

```diff
@@ -1178,6 +1190,24 @@ Vision.findImages_ = function(images, callback) {
       return;
     }
 
+    // File is a URL.
+    if (image.indexOf('http') === 0) {
```

Comment:
Probably splitting hairs, but would this be better as a regex to verify the placement of http? (e.g. `^http`)